cognite_toolkit/_cdf_tk/resource_classes/dataset.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import Field
+
+from .base import ToolkitResource
+
+
+class DataSetYAML(ToolkitResource):
+    external_id: str
+    name: str | None = Field(
+        default=None,
+        description="The name of the data set.",
```

Comment:
At this stage we are not using description, so you do not have to include it. But it doesn't hurt that you have it.